test/envtest/adminapimock.go

```diff
@@ -1,40 +1,64 @@
 package envtest
 
 import (
-	"encoding/json"
+	"io"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 )
 
+const dblessConfig = `{
+	"version": "3.3.0",
+	"configuration": {
+		"database": "off",
+		"router_flavor": "traditional",
+		"role": "traditional",
+		"proxy_listeners": [
+			{
+				"backlog=%d+": false,
```

Comment:
Is this expected to contain the format strings ? 🤔

Reply:
Yup, that's taken directly straight from Kong root response. Here are go-kong's API mappings for it https://github.com/Kong/go-kong/blob/c7d04455bcfbe36b655972e232b5e402144ab7cb/kong/listeners.go#L16C2-L16C2.